ui/lib/core/addon/helpers/format-number.js

```diff
@@ -1,11 +1,11 @@
 import { helper } from '@ember/component/helper';
 
```

Comment:
Originally was going to add `parseInt` to properly format a string value passed in, but the _only_ reason to do so would be if a string was passed in that wasn't formatted properly (i.e. "2141" instead of "2,141"). It ended up being more trouble than it was worth, for an unlikely edge case. SO, I updated the variable name and went on my merry way.